src/engine/Server.cpp

```diff
@@ -631,7 +631,6 @@ boost::asio::awaitable<void> Server::processQuery(
     auto& qet = queryExecutionTree.value();
     qet.isRoot() = true;  // allow pinning of the final result
     qet.recursivelySetTimeoutTimer(timeoutTimer);
-    qet.getRootOperation()->createRuntimeInfoFromEstimates();
```

Comment:
Was this redundant and is now wrong because of the `shared_ptr` semantics?

Reply:
This is still redundant because the root operation always calls this function on getResult()

Reply:
The let's delete it!